package.json

```diff
@@ -31,7 +31,8 @@
     "@skyux/indicators": "^4.0.0"
   },
   "dependencies": {
-    "intl-tel-input": "14.1.0"
+    "intl-tel-input": "14.1.0",
+    "lodash.orderby": "4.6.0"
```

Comment:
What's this?

Reply:
Oops... Not really sure why that got in there. Removed!